### ISSUE TITLE
disassemble_circuits() suggested in qobj_to_circuits.py DeprecationWarning doesn't exist

### ISSUE THREAD

**Issue by matteoacrossi** [Qiskit/qiskit-terra#2334]
### Information

- **Qiskit Terra version**: 0.8.0
- **Python version**: 3.7.2
- **Operating system**: macOS

`qobj_to_circuits` gives the following `DeprecationWarning`:

```python
.../qiskit/converters/qobj_to_circuits.py:34: DeprecationWarning: qiskit.converters.qobj_to_circuit() is deprecated and will be removed in Qiskit Terra 0.9. Please use qiskit.compiler.disassemble_circuits() to convert a qobj to list of circuits.
```

but `qiskit.compiler.disassemble_circuits()` doesn't exist.

### What is the current behavior?
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'disassemble_circuits' from 'qiskit.compiler' (/Users/matteo/Work/projects/ibmq/env/lib/python3.7/site-packages/qiskit/compiler/__init__.py)
```

### Steps to reproduce the problem
1. Installed qiskit in a new python virtualenv with `pip install qiskit`
2.  `from qiskit.compiler import disassemble_circuits`

```
>>> qiskit.__qiskit_version__
{'qiskit': '0.10.0', 'qiskit-terra': '0.8.0', 'qiskit-ignis': '0.1.1', 'qiskit-aer': '0.2.0', 'qiskit-ibmq-provider': '0.2.1', 'qiskit-aqua': '0.5.0'}
```

### What is the expected behavior?
If a function is deprecated, and the warning suggests to use a new function, this function should exist in the current release.

### Suggested solutions

Implement the function or change the deprecation warning.

**Comment by ajavadia** [comment]
Sorry there seems to be a mistake in the deprecation message. For now please use
```from qiskit.assembler import disassemble```

@mtreinish I think `disassemble` should be added under the `qiskit.compile` namespace.

**Comment by jaygambetta** [comment]
yeah this is a bug we did not update the disassemble warning when we changed API. This was probably my fault.

**Comment by mtreinish** [comment]
It was when I originally added the function (and deprecation message) in #2137 but it looks like that was changed in #2244 right before the release without updating the deprecation message.

**Comment by jaygambetta** [comment]
so yeah my fault :-(

**Comment by ajavadia** [comment]
fixed by #2350
